### DISCUSSION TITLE
DietPi-Software | Logitech Media Server: Nightly builds for Bookworm support

### DISCUSSION THREAD

**Discussion by SteveInWA** [MichaIng/DietPi#6847] [Software Request] [answered]
The latest build of Logitech Media Server (8.3.2 as of today) now includes Perl 5.36 support for Linux aarch64, armv7.  8.3.1 added Perl 5.36 for Linux x86/x64.

[https://htmlpreview.github.io/?https://github.com/Logitech/slimserver/blob/public/8.3/Changelog8.html](https://htmlpreview.github.io/?https://github.com/Logitech/slimserver/blob/public/8.3/Changelog8.html)

**Comment by Joulinar** [comment]
Best to my knowledge, 8.3.2 is still a nightly build, right? https://lms-community.github.io/lms-server-repository/ and https://github.com/Logitech/slimserver/tags

**Comment by SteveInWA** [comment]
That's how they have distributed the software for years.  It is so stable
that new builds are mainly maintenance to fix some small issue, like the
PERL compatibility problem.  It is safe and reasonable to simply install
the latest build.

*--Steve*

**Comment by MichaIng** [comment]
Hmm, we get the link from here, but this shows the latest stable build, hence v8.3.1: https://www.mysqueezebox.com/update/?version=8.3&geturl=1&os=debarm

Based on the info from the lms-community page, it seems reasonable to use the nightly builds, but I have no idea how to catch the latest one of them. Also scraping the page itself is no easily possible, since there is no difference in the HTML lines between stable, nightly and development builds. Some API like the one we use would be great.

**Comment by SteveInWA** [comment]
Hmm; I see your dilemma.  I asked over on the Squeezebox (LMS) forum for a
solution:

https://forums.slimdevices.com/forum/user-forums/logitech-media-server/1666070-will-lms-8-3-2-transition-to-latest-release-status

*--Steve*

**Comment by MichaIng** [comment, marked as answer]
Implemented: https://github.com/MichaIng/DietPi/commit/57cdcf1
